.craft.yml

```diff
@@ -0,0 +1,9 @@
+minVersion: 0.0.1
+github:
+  owner: getsentry
+  repo: infra-event-notifier
+changelogPolicy: auto
+targets:
+  - name: pypi
```

Comment:
not sure if it really matters, but i think `github` is also on this list for the other libs i've seen at sentry

Reply:
github would create a github release. It is not really required for our packaging system. Up to you.